src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -410,8 +407,29 @@ public SnapshotAccessRequestResponse approveRequest(UUID id) {
 
   private SnapshotAccessRequestResponse convertModelToApiResponse(
       SnapshotAccessRequestModel model) {
-    return model.toApiResponse(
-        snapshotBuilderSettingsDao.getBySnapshotId(model.sourceSnapshotId()));
+    return model.toApiResponse();
```

Comment:
Now that this method is trivial, I think it can be inlined, so the caller would call `model.toApiRequest()` directly.